### ISSUE TITLE
Rain Last Hour

### ISSUE THREAD

**Issue by thomas70** [briis/hass-weatherflow2mqtt#191]
### New Feature

Would really like a rain_last_hour sensor

**Comment by briis** [comment]
Not that easy to do correctly, as the UDP API only delivers last minutes rain accumulation. 
It can be done manually by creating a few helpers and an automation, if you need it now.

**Comment by pgneir** [comment]
I too wanted periodic totals. Like last 4 hours, last 24 hours, last week, last month. The daily totals weren't useful for me. What I want is each one minute rain accumulation from the hub. Can I assume that I can get that by dividing your rain_rate by 60?? 
   I solved my problem by using the SQL integration. I can get everything I want. Here is an example:
"select round(sum(state)/60,2) as rain_24 from states where entity_id='sensor.tempest_st_nnnnnnnn_rain_rate' and last_updated > datetime('now','-24 hours');"

**Comment by briis** [comment]
> Can I assume that I can get that by dividing your rain_rate by 60??

Yes you can. The device reports the last minutes accumulated rain as the raw data, and that is then multiplied to get rain_rate per hour.
